charts/etcd/values.yaml

```diff
@@ -3,6 +3,10 @@ uid: uuid-of-etcd-resource
 serviceName: test
 configMapName: test
 
+pdb:
+  name: test-pod-disruption-budget
```

Comment:
Any reason why we need a different name? Otherwise, I propose to to remove this and use `name` instead.